public-cloud/azure/ixia-c-dpdk-azure-mellanox/configs/docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+services:
+ 
+  controller:
+    image: ghcr.io/open-traffic-generator/keng-controller:1.3.0-2
```

Comment:
I think we should use the latest controller & TE version - it's easier to address any potential issues and it has new features like OPT_ADAPTIVE_CPU_USAGE argument.  https://ixia-c.slack.com/archives/C021DU5026R/p1723056439876559